services/UsersService.py

```
@@ -79,12 +79,18 @@ class LinkCodeforcesResponse(Enum):
         ERROR_USER_NOT_FOUND = 1
         ERROR_INCORRECT_FIRST_NAME = 2
+        ERROR_ALREADY_USED_ACCOUNT = 3
 
     def link_codeforces(self, handle: str, user_id: str) -> LinkCodeforcesResponse:
         response = requests.get(f"https://codeforces.com/api/user.info?handles={handle}&checkHistoricHandles=false")
         response_dict = response.json()
         if response_dict['status'] != "OK":
             return self.LinkCodeforcesResponse.ERROR_USER_NOT_FOUND
+        is_registered = self.databaseWorker.find_one('users', {
+            "codeforce.handle" : handle
+        }, ray_id)
```

Ruff (0.8.2)

Undefined name `ray_id`

(F821)